discord/gateway.py

```diff
@@ -553,6 +571,15 @@ def from_client(cls, client):
         }
 
         yield from ws.send_as_json(identify)
+
+        try:
+            # Poll for useless HELLO event, then READY
+            yield from asyncio.wait_for(ws.poll_event(), timeout=60, loop=client.loop)
+            yield from asyncio.wait_for(ws.poll_event(), timeout=60, loop=client.loop)
+        except asyncio.TimeoutError:
+            log.warn("timed out waiting for voice client READY")
```

Comment:
u should also attempt to close the `ws` at this point. otherwise u are leaking the fd.